GVFS/GVFS.UnitTests/Mock/Common/MockTracer.cs

```diff
@@ -132,6 +132,10 @@ public TimeSpan Stop(EventMetadata metadata)
             return TimeSpan.Zero;
         }
 
+        public void SetGitCommandSessionId(string sessionId)
+        {

```

Comment:
Just asking.  Should we do something here?

Reply:
Let me look to see if there are some tests where we could validate that this is getting called and set the the right thing.

Reply:
I was able to add to the `GVFSLockTests.cs` but used Moq instead since it was easier.  The other place would be for the `InProcessMount` class which doesn't have any unit tests for it currently so would be a little bigger undertaking to get that one.  So I am going to leave this as is for now since nothing else needs it for testing.  Thanks!